react/ProductDetails.js

```diff
@@ -4,12 +4,8 @@ import {
   mapObjIndexed,
   mergeDeepRight,
   path,
-  compose,
-  flip,
+  pathOr,
```

Comment:
This line is breaking a test.

<img width="994" alt="Screen Shot 2019-04-16 at 15 42 35" src="https://user-images.githubusercontent.com/27777263/56235704-5160c700-605e-11e9-9839-3e9181512653.png">


This is from Travis log